src/app/login/page.tsx

```diff
@@ -0,0 +1,156 @@
+'use client';
+
+import React, { useState, type JSX } from 'react';
+import '../../styles/Login.css';
+
+const TEST_ACCOUNT = {
+  email: 'username',
+  password: 'change1234'
+};
+
+const login = async (email: string, password: string): Promise<{ success: boolean; message?: string }> => {
+  await new Promise(resolve => setTimeout(resolve, 1000));
+  
+  if (email === TEST_ACCOUNT.email && password === TEST_ACCOUNT.password) {
+    return { success: true };
+  } else {
+    return { success: false, message: 'Invalid email or password' };
+  }
+};
+
+function LoginPage(): JSX.Element {
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const [loading, setLoading] = useState<boolean>(false);
+  const [emailError, setEmailError] = useState<string>('');
+  const [passwordError, setPasswordError] = useState<string>('');
+  const handleLogin = async (): Promise<void> => {
+
+    setEmailError('');
+    setPasswordError('');
+    let hasError = false;
+
+    if (!email) {
+      setEmailError('Employee ID is required!');
+      hasError = true;
+    }
+
+    if (!password) {
+      setPasswordError('Password is required!');
+      hasError = true;
+    }
+
+    if (hasError) {
+      return;
+    }
+
+    setLoading(true);
+
+    try {
+      const result = await login(email, password);
+      
+      if (result.success) {
+        if (typeof window !== 'undefined') {
+          sessionStorage.setItem('isLoggedIn', 'true');
+          sessionStorage.setItem('userEmail', email);
+        }
+        
+        alert('Login Successful!');
+        window.location.href = "/";
+      } else {
+        setEmailError('Invalid Employee ID!');
+        setPasswordError('Invalid Password!');
+      }
+    } catch (error) {
+      alert('Error: Something went wrong. Please try again. ' + String(error));
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>): void => {
+    if (e.key === 'Enter') {
+      handleLogin();
+    }
+  };
+
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setEmail(e.target.value);
+    if (emailError) setEmailError('');
+  };
+
+  const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setPassword(e.target.value);
+    if (passwordError) setPasswordError('');
+  };
+
+  return (
+    <div className="background">
```

Comment:
```suggestion
    <div className={style.background}>
```
implement this to other component as well but I suggest you to just use tailwind css instead